docs/src/tutorial/custom-data.rst

```diff
@@ -0,0 +1,158 @@
+
+.. code-block:: yaml
+
+   inputs:
+     - name: reference_data
+       metadata: https://data.nextstrain.org/files/ncov/open/reference/metadata.tsv.xz
+       sequences: https://data.nextstrain.org/files/ncov/open/reference/sequences.fasta.xz
+     - name: custom_data
+       metadata: data/custom.metadata.tsv
+       sequences: data/custom.sequences.fasta
+
+   refine:
+     root: "Wuhan-Hu-1/2019"
+
+   builds:
+     custom-build:
+       title: "Build with custom data and example data"
+       auspice_config: ncov-tutorial/auspice-config-custom-data.json
+
+This is the same as the previous file, with some additions:
+
+1. A second input for the custom data, referencing the metadata and sequences files downloaded from GISAID.
+2. A ``builds`` section that defines one output :term:`docs.nextstrain.org:dataset` using:
+
+   1. A custom name ``custom-build``
+   2. A custom title ``Build with custom data and example data``
+   3. An Auspice config file with the contents:
+
+      .. code-block:: json
+
+         {
+           "colorings": [
+             {
+               "key": "custom_data",
+               "title": "Custom data",
+               "type": "categorical"
+             }
+           ],
+           "display_defaults": {
+             "color_by": "custom_data"
+           }
+         }
+
+      This JSON does two things:
+
+      1. Create a new coloring ``custom_data`` which reflects a special metadata column generated by the ncov workflow. Each data input produces a new final metadata column with categorical values ``yes`` or ``no`` representing whether the sequence was from the input.
+      2. Set the default Color By as the new ``custom_data`` coloring.
+
+   .. note ::
+
+      **Build** is a widely used term with various meanings. In the context of the ncov workflow, the ``builds:`` section defines output :term:`datasets <docs.nextstrain.org:dataset>` to be generated by the workflow (i.e. "build" a dataset).
+
+Visualize the results
+---------------------
+
+Run this command to start the :term:`docs.nextstrain.org:Auspice` server, providing ``auspice/`` as the directory containing output dataset files:
+
+.. code:: text
+
+   nextstrain view auspice/
+
+Navigate to ``http://127.0.0.1:4000/ncov/custom-build``. The resulting :term:`docs.nextstrain.org:dataset` should have similar phylogeny to the previous dataset, with additional sequences:
+
+.. figure:: ../images/dataset-custom-data-highlighted.png
+   :alt: Phylogenetic tree from the "custom data" tutorial as visualized in Auspice
+
+
+1. The custom dataset name ``custom-build`` can be seen in the dataset selector, as well as the dataset URL.
+2. The custom dataset title can be seen at the top of the page.
+3. The custom coloring is used by default. You can see which sequences are from the custom data added in this tutorial.
+
+   .. note::
+
+      You may not see all 10 custom sequences - some can be filtered out due to quality checks built into the ncov workflow.
```

Comment:
(Adding here so the discussion can be threaded).

We have a single page in the Guide section named "Preparing your data" which acts a bit like a tutorial as well as explaining the formatting of the files as well as explaining how to download from GISAID. I would suggest we:

1. Add a section at the end of this tutorial ("Run the workflow using custom data") saying that you can easily replace the GISAID sample here with your own private sequences, but to do this the metadata & fasta must be in a specific format (which here GISAID manages for us). This can then point to a page labelled "Preparing your data"
2. Modify the page "Preparing your data" to only consider the FASTA / TSV formatting. All the GISAID information, as well as how to run the analysis, is covered in the tutorials so it shouldn't be repeated in that guide.

We only have one guide... will we add more? If not, let's shift that page to the "Reference material" section. cc @huddlej for thoughts.

Reply:
Started this: c03dcd5fc1e3d006d1e20c760f2ac320ee84d284

Reply:
Looks great!

I rearranged the sidebar in https://github.com/nextstrain/ncov/pull/906 which makes this section more prominant. 

I also think we need to add this as a hint / note in the "Run the workflow using custom data" page, as it seems natural to want to go from that tutorial to using your own (non-GISAID) data.

Reply:
Done: https://github.com/nextstrain/ncov/blob/65084a99402e30ef0401501b7d0bfd391d9c9bfa/docs/src/tutorial/custom-data.rst?plain=1#L83-L85